src/System.IO.FileSystem/tests/FileStream/ctor_str_fm_fa_fs_buffer_fo.cs

```diff
@@ -25,19 +25,58 @@ public void InvalidFileOptionsThrow()
             Assert.Throws<ArgumentOutOfRangeException>("options", () => CreateFileStream(GetTestFilePath(), FileMode.Create, FileAccess.ReadWrite, FileShare.Read, c_DefaultBufferSize, ~FileOptions.None));
         }
 
-        [Fact]
-        public void ValidFileOptions()
+        [Theory]
+        [InlineData(FileOptions.None)]
+        [InlineData(FileOptions.DeleteOnClose)]
+        [InlineData(FileOptions.Encrypted)]
+        [InlineData(FileOptions.RandomAccess)]
+        [InlineData(FileOptions.SequentialScan)]
+        [InlineData(FileOptions.WriteThrough)]
+        [InlineData((FileOptions)0x20000000)] // FILE_FLAG_NO_BUFFERING on Windows
```

Comment:
Yeah, we just ignore it.  I considered mapping it to O_DIRECT, but that's not portable and it comes with additional requirements, plus this option isn't actually part of the surface area.